### ISSUE TITLE
Docstring not rendering properly inside table

### ISSUE THREAD

**Issue by theabhirath** [lorenzoh/FeatureRegistries.jl#6]
I've been trying to get FeatureRegistries.jl up for [Metalhead.jl](https://github.com/FluxML/Metalhead.jl/pull/153) and I've hit a bit of a snag. While trying to get the docstring to display, it gets cropped out of the table:

<img width="838" alt="Screenshot 2022-05-14 at 7 43 11 AM" src="https://user-images.githubusercontent.com/74202102/168407061-e84dc4f4-8556-4b38-9e17-edf9d2e361ec.png">

Is there a way to format this by toggling an option in the table so that the newlines render at the appropriate places and the entire docstring is visible? Or is this something I will have to tweak in the markdown for the docstring manually?

**Comment by lorenzoh** [comment]
Cool to see you try this out!

The text rendering is by default limited to 40 characters per column:
https://github.com/lorenzoh/FeatureRegistries.jl/blob/ff0b1b49fd472fcc33c277fe04fdcdca4b63b147/src/printing.jl#L125-L138

I should make that configurable and allow passing kwargs for `pretty_table` to `Registry` 👍 

As an aside, while I love the way the docstring looks inside the table, I wonder if cropping it vertically to allow more rows to display (at least in text display) may make it easier to find entries. You can always see all of a row's attributes using `registry[id]`. I think this would also require making the text display more configurable on my side, though. I'm keeping this  open until I get to that :)

**Comment by theabhirath** [comment]
Yeah, it's a great package, the display looks very pretty! The kwargs for `pretty_table` via `Registry` should definitely help. I did think about having more rows to display, but I noticed that `show(::Registry)` shows all the rows. Seeing more rows would definitely not hurt though
